container/tabs.go

```diff
@@ -79,6 +79,7 @@ type baseTabs interface {
 
 	items() []*TabItem
 	setItems([]*TabItem)
+	scope() fyne.Widget
```

Comment:
sorry, I don't really know why this is needed. If it really is needed, can we add a comment explaining why (and what)?

Reply:
It is because of the baseTabs is not a widget type, it represents the commonality of two types of widget but is itself not a widget.

Reply:
Added a note

Reply:
If both the baseTabs implementations (AppTabs and DocTabs) are widgets, why not add `fyne.Widget` to the baseTabs interface and then the scope() is not needed?

Reply:
I didn't think of that as I was using *BaseWidget before where that would not work. I'll try with the interface now it's a possibility thanks.

Reply:
Fixed, nice suggestion thanks.